src/components/Quote.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import '../styles/quote.css';
+
+const Quote = () => {
+  const [quote, setQuote] = useState('');
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+      headers: { 'X-Api-Key': 'abI0ZFGUK8RjoHhoF9KItg==QfJBvXHJhb0V6EUy' },
```

Comment:
- [ ] [Optional] Well done on your project! 🚀 I noticed that the ninjas API key is currently visible in your code, which may pose security risks. API keys, especially those granting access to sensitive resources, should be kept private to prevent unauthorized access and potential misuse. Storing API keys directly in code, especially in publicly accessible repositories (like open-source projects on GitHub), exposes them to potential attackers.

I strongly recommend using environment variables to manage your API keys. Keep up the excellent work! If you have any questions or need guidance on implementing environment variables, feel free to ask. 😊🌟